docs/unified-test-documentation/dasharo-security/200-tpm-support.md

```diff
@@ -404,3 +404,374 @@ Example output:
 ```text
 SpecVersion=2.0
 ```
+
+## TPM003.001 PCRREAD Function (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that PCRREAD function works properly
```

Comment:
In my opinion, in the test description section, we should describe at least in a few words, what the function does.